### DISCUSSION TITLE
Remove default clean shortcut

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-m2e/m2e-core#1453] [Ideas]
While working on the new feature to show launch configs I noticed that we already provide a lot of items so one can easily run out of accelerator keys.

I therefore try to reduce these and cam across the `clean` action, I think it can be removed for the following reasons:

1. in almost all cases its easier to just delete the `target` folder directly then clicking through the menus
2. we probably better should  link this to `Project > Clean ...` action that is triggering the m2e builder with a request to clean
3. if a user requires a clean shortcut one could be created quite easy

**Comment by HannesWell** [comment]
I use the shortcut to launch a clean-build quite often, when I want to remove all the artifacts from a previous build in a projects of a git-repo.
Then I can just select the root, run the maven clean shortcut and I'm done.

> we probably better should link this to Project > Clean ... action that is triggering the m2e builder with a request to clean

This would be a bit different compared to now, because now I can clean my workspace without deleting e.g. the 'expensive' to build artifacts like products or a p2-repo. But on the other hand, if one wants to keep them, then one should simply not clean them.

And it would have the advantage that m2e can claim in its default lifecycle mappings that it handle the maven-clean-plugin, which is currently to only reason why the old m2e-pde connector is still suggested (who knows why the m2e-pde connector claimed to handle this).

When we do this, m2e should fully read the config of the maven-clean-plugin and should also consider alternative/additional locations to clean.
